src/hooks/usePropsGetters/index.ts

```diff
@@ -137,17 +137,17 @@ const usePropsGetters = (
 
   const getQuizImageProps: GetQuizImageProps = useCallback(
     () => ({
-      src: quizApiState.quizCurrentQuestion?.next_question?.images?.primary_url,
-      alt: quizApiState.quizCurrentQuestion?.next_question?.images?.primary_alt,
+      src: quizApiState.quizCurrentQuestion?.next_question?.images?.primary_url || undefined,
+      alt: quizApiState.quizCurrentQuestion?.next_question?.images?.primary_alt || undefined,
     }),
     [quizApiState.quizCurrentQuestion]
   );
 
   const getSelectQuestionImageProps: GetSelectQuestionImageProps = useCallback(
     (option) => ({
       className: 'cio-question-option-image',
-      src: option?.images?.primary_url,
-      alt: option?.images?.primary_alt,
+      src: option?.images?.primary_url || undefined,
```

Comment:
src and alt on img html element accept only type string or undefined and not null